topics/programability.md

```diff
@@ -0,0 +1,93 @@
+# Redis Programability
+Programability in Redis allows the user to send functionality (scritps) into Redis that will run on the sever where the data leaves.
+This can reduce network hops and improve overal perfromance.
+It can also enrich the API provided by Redis with a user define API,
+and allow implementing a schema over Redis data structures (updates multiple keys together, keep keys in sync).
+
+Currently, scripts can only be written in [Lua](https://www.lua.org/).
+Redis runs an embeded Lua 5.1 interpreter that is used to run the user scripts.
+The full Lua API provided to the user is documented on [Redis Lua API](/topics/lua).
+
+In addition, Redis provides two ways to run the scripts:
```

Comment:
In addition to what? (In addition to simply providing a specification is not additional enough. :D)

```suggestion
Redis provides two ways to run scripts:
```